Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -162,6 +176,10 @@ public struct WorldScaleGeoTrackingSceneView: View {
                 .padding(8)
                 .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
         }
+        .onChange(of: viewModel.scene.basemap?.loadStatus) { loadStatus in
+            guard loadStatus == .loaded else { return }
```

Comment:
Add some comment about what is the expected visual clue when the basemap first loads.